### DISCUSSION TITLE
The title can be entered, but the content cannot be entered.

### DISCUSSION THREAD

**Discussion by surinara** [outline/outline#6569] [Self Hosting]
**To Reproduce**
1. Outline received the latest source git clone and executed it.
2. After adding a new page, enter a title.
3. Enter the Enter or tab in the title part, but it does not go over to the content part.
4. When you click the mouse, the focus does not shift.
5. Only the title is in edit mode and the content does not seem to be edit mode.

**Expected behavior**
The title can be entered, but the content cannot be entered.

**Screenshots**
![image](https://github.com/outline/outline/assets/160732515/1f9baf05-004e-4a92-9391-7cab77a864d9)

**Outline (please complete the following information):**
- Install : self-hosting
- Version : 7c03963

**Desktop (please complete the following information):**
 - OS : Windows 11
 - Browser : Edge
 - Version : 121.0.2277.128

**Comment by linear** [comment]
<p><a href="https://linear.app/outline/issue/OLN-269/the-title-can-be-entered-but-the-content-cannot-be-entered">OLN-269 The title can be entered, but the content cannot be entered.</a></p>

**Comment by surinara** [comment]
I solved it. My problem was nginx setup problem. I found the solution in the answer in the link below.
https://github.com/outline/outline/discussions/3677
> proxy_set_header Connection "Upgrade";
